test/functional/fixtures/page-js-errors/test.js

```diff
@@ -56,11 +66,226 @@ describe('Should ignore an js-error on the page if the skipJsErrors option is se
         return runTests('./testcafe-fixtures/error-after-click-test.js', 'Click button', { skipJsErrors: true });
     });
 
-    it ('unhandled Promise rejection', () => {
+    it('unhandled Promise rejection', () => {
         return runTests('./testcafe-fixtures/unhandled-promise-rejection-test.js', 'Click button',
             {
                 skipJsErrors: true,
                 only:         'chrome',
             });
     });
 });
+
+const expectFailAttempt = (errors, expectedMessage) => {
+    Object.values(errors).forEach(err => {
+        const error = castArray(err);
+
+        expect(error[0].message || error[0]).contains(expectedMessage);
+    });
+};
+
+// TODO: fix tests for Debug task
+(experimentalDebug ? describe.skip : describe)('[Regression](GH-2775) SkipJSErrors API', () => {
```

Comment:
Rename to `(experimentalDebug ? describe.skip : describe)('Customize SkipJSErrors (GH-2775))', () => {`